packages/design-system/src/components/OcBreadcrumb/OcBreadcrumb.vue

```diff
@@ -255,6 +263,13 @@ export default defineComponent({
       hiddenItems.value.length >= 1 ? unref(hiddenItems)[unref(hiddenItems).length - 1] : { to: {} }
     )
 
+    const translate = (breadcrumbIndex: Number) => {
+      if (!props.representsFolderStructure) {
```

Comment:
Could you use the `isStaticNav` prop on the breadcrumb item instead so that you don't need to introduce the new component prop? Or check for absence of an `id` on the breadcrumb item?

Reply:
I'll check that :) is is unfortunately not sufficient